elide-integration-tests/src/test/java/example/CreateButNoRead.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import com.yahoo.elide.annotation.CreatePermission;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.ReadPermission;
+import com.yahoo.elide.security.ChangeSpec;
+import com.yahoo.elide.security.RequestScope;
+import com.yahoo.elide.security.checks.OperationCheck;
+import com.yahoo.elide.security.checks.prefab.Role;
+
+import java.util.Optional;
+import java.util.Set;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+
+/**
+ * A model intended to be ONLY created and read, but never updated
+ */
+@Include(rootLevel = true)
+@Entity
+@CreatePermission(any = {Role.ALL.class})
+public class CreateButNoRead extends BaseId {
+    private Set<CreateButNoReadChild> otherObjects;
+
+    @OneToMany(fetch = FetchType.LAZY)
+    @ReadPermission(any = {NOREAD.class})
```

Comment:
@DeathByTape This read check is deferred on patchext add for the newly created child object during post-processing.